spanner/tests/_fixtures.py

```diff
@@ -47,6 +47,7 @@
     name STRING(16),
     tags ARRAY<STRING(16)> )
     PRIMARY KEY (id);
+CREATE INDEX name ON contacts(first_name, last_name);
```

Comment:
This change (and the ones below which accompany it) should be split out into a separate PR:  there is no point in cluttering the rest of them with the noise.